common_sls.py

```diff
@@ -33,9 +33,10 @@
 #          20. GetFsSpace : Check how much free space is available on boot disk
 #          21. SetMinFree : Sets min_free_kbytes and swappiness as per SLS requirement
 #          22. CreateFS : Creates Filesystem/LVM on IO disks
-#          23. OOMKill : Kills SLS related processes if free memory goes below 10% of total memory
-#          24. ParseScenFile : Parses scenario file if SLS is executed with -r option
-#          25. GetSuiteIterations : Finds to which suite a test belongs to and decides how many iterations a test should be executed
```

Comment:
Please add back the above three lines

Reply:
Hi Manju, 
Those 3 lines are there already. I added one comment for "CreatePMEMFS" description before "OOMKill". 
Changes done are as follows: 
#          23. CreatePMEMFS : Creates pmem namespaces and filesystem required for IO test
#          24. OOMKill : Kills SLS related processes if free memory goes below 10% of total memory
#          25. ParseScenFile : Parses scenario file if SLS is executed with -r option
#          26. GetSuiteIterations : Finds to which suite a test belongs to and decides how many iterations a test should be executed